### ISSUE TITLE
some troubles

### ISSUE THREAD

**Issue by shadowjades** [gusanagy/Diffusion-Underwater-Image-Enhancement#4]
Dear Authors,

I've been learning and reproducing the project you posted recently, and I've been following the training process in the project in three phases in sequence. But in practice, I tried to train several times in a row, but I never got the desired results.

In the first stage of training (e.g., Charbonnier + MS-SSIM), the overall results of the model were acceptable, although the output image quality was low. However, from the second stage, the loss still had a downward trend at the beginning of training, but soon it suddenly soared to 2~3 and continued to decrease, and the image output by the model became pure noise. After entering the third stage, the loss also did not decrease significantly, and the result was still a colored noise map.

The code in the project involves multiple dataset names (such as UIEB, LSUI, TM-DIED, HDR+, etc.), but it doesn't make it clear which dataset or datasets will be used for actual training. Therefore, I would like to ask:

Which dataset did you choose to use when you formally trained your model?

Is it convenient to make a copy of the pre-trained model you trained public, so that I can better learn and understand the model structure and inference effect?

I've been trying to train for almost a week but have never been able to get consistent results, so I'd love to have some guidance from you. Thank you very much!

Best wishes
